jsuite/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# pull official base image
+FROM python:3
```

Comment:
Careful here, now we have the CI running on 3.9 but I think this doesn't necessarily point to 3.9 as well (3.10.5 in my container).